src/types.ts

```diff
@@ -4,20 +4,40 @@
  */
 
 /* eslint-disable @typescript-eslint/no-empty-interface */
-
-import type * as express from 'express';
 import type * as http from 'http';
 import type * as httpProxy from 'http-proxy';
 import type * as net from 'net';
 import type * as url from 'url';
 
-export interface Request extends express.Request {}
-export interface Response extends express.Response {}
+export type Request = http.IncomingMessage;
+export type Response = http.ServerResponse;
```

Comment:
👀 this is the fist most significant change in this PR

Reply:
these could be dropped outright, and just use `http.*` everywhere. i kept them in, to reduce churn. but `Request`/`Response` are now just aliases